ui/apps/platform/src/Containers/NetworkGraph/external/EntityDetails.tsx

```diff
@@ -167,7 +168,18 @@ function EntityDetails({
                                             const { deployment, id } = entity;
 
                                             const onEntitySelect = () => {
-                                                onNodeSelect(id);
+                                                onNodeSelect(
```

Comment:
Independent of question about coupling via `buildSearchQuery` function, does it become possible (because of this effort) to build an address and use an ordinary link instead of call an imperative callback function?

Reply:
Unfortunately I don't think so because of all the side effects happening inside of the `onNavigate` function at the topology component.